### ISSUE TITLE
CDN issue

### ISSUE THREAD

**Issue by simkoG** [lucide-icons/lucide#903]
When you add Lucide via the [official CDN](https://lucide.dev/docs/lucide#cdn) there is a warning within DevTools that the "source map could not loaded". 

## Prerequisites

* Version: latest
* Are you running from source/main: import from CDN
* Operating system: Windows 10
* Bits: 64 bits

## Step to reproduce

1) Add `<script src="https://unpkg.com/lucide@latest"></script>` before the closing `</body>` tag in a HTML page. 
2) Check the browser's console log. 

### Actual behavior

You can see the icons but there is a warning message. 

## Any message or error

```
DevTools failed to load source map: Could not load content for https://unpkg.com/lucide.min.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```

## Resources

* https://codepen.io/simkog/pen/WNKGZro

![image](https://user-images.githubusercontent.com/10430702/210541107-a55cee14-8cf5-413b-8979-0c11a23da4a9.png)

**Comment by karsa-mistmere** [comment]
This is not really something we at Lucide can do much about, I'm afraid.

Since `https://unpkg.com/lucide@latest` redirects to `https://unpkg.com/lucide@0.104.0/dist/umd/lucide.min.js`, the current `sourceMappingURL` should in theory work just fine, since the source maps are there; unfortunately neither Chrome nor Firefox seems to respect this redirect when querying the JS maps.

We could replace references to `https://unpkg.com/lucide@latest` with `https://unpkg.com/lucide@latest/dist/umd/lucide.min.js` in our documentation, although since `https://unpkg.com/lucide@latest` is the recommended _production_ CDN URL, this does not seem necessary, since source maps are typically not used in production.

TLDR: I suggest that you use `https://unpkg.com/lucide@latest/dist/umd/lucide.min.js` or `https://unpkg.com/lucide@latest/dist/umd/lucide.js` instead of `https://unpkg.com/lucide@latest`.
